README.md

```diff
@@ -26,6 +26,9 @@ We are iterating and looking for feedback and collaboration, so please [**let us
  - Predictions
  - Datastore
  - Hub Events (Listening to the Amplify events)
```

Comment:
A bit off topic, but we currently listen for Auth Hub events... can we clarify that Hub events would be needed for storage and analytics, but are present for Auth?

Reply:
done

 - Hub Events for Storage and Analytics (but are implemented on Auth)